docs/source/reference/volumes.rst

```diff
@@ -110,6 +110,127 @@ You can also check the volumes in the SkyPilot dashboard.
     :align: center
     :width: 80%
 
+Filesystem volume examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section demonstrates how to configure and use distributed filesystems as SkyPilot volumes. We'll cover popular options like `JuiceFS <https://juicefs.com/docs/community/introduction/>`_ (a cloud-native distributed filesystem) and `Nebius shared file system <https://docs.nebius.com/compute/storage/types#filesystems>`_ (a high-performance shared storage solution).
+
+
+.. tab-set::
+
+    .. tab-item:: JuiceFS
+        :sync: juicefs-tab
+
+        To use JuiceFS as a SkyPilot volume:
+
+        1. **Install the JuiceFS CSI Driver** on your Kubernetes cluster. Follow the official `installation guide <https://juicefs.com/docs/csi/getting_started>`_ for detailed instructions.
+
+        2. **Verify the storage class** - Confirm that the ``juicefs-sc`` storage class has been created successfully:
+
+        .. code-block:: console
+
+          $ kubectl get storageclass
+          NAME           PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+          juicefs-sc     csi.juicefs.com     Retain          Immediate           false                  10m
+
+        .. note::
+           If the ``juicefs-sc`` storage class is not available, refer to the `JuiceFS storage class creation guide <https://juicefs.com/docs/csi/guide/pv/#create-storage-class>`_ to set it up.
+
+        3. **Create a JuiceFS volume** using SkyPilot's volume configuration:
+
+        .. code-block:: yaml
+
+          # juicefs-volume.yaml
+          name: juicefs-pvc
+          type: k8s-pvc
+          infra: k8s
```

Comment:
do we need to specify this? it seems quite duplicated to the `type`

Reply:
Yes for now, I created an issue https://github.com/skypilot-org/skypilot/issues/6416 to follow up this.